sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -124,6 +124,9 @@ final class CastExprMeta[INPUT <: CastBase](
           case (fromChild, toChild) =>
             recursiveTagExprForGpuCheck(fromChild.dataType, toChild.dataType, depth + 1)
         }
+      case (ArrayType(element_type, _), StringType) =>
```

Comment:
nit `element_type` does not follow the convention. Could we make it `elementType`?

Reply:
Thank you, I will fix it